tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/views/alerts/alerts_component.ng.html

```diff
@@ -24,7 +24,12 @@
     </div>
   </div>
   <div class="alerts-breakdown-container">
-    <div *ngFor="let breakdown of alertsBreakdown;">
+    <div
+      *ngFor="let breakdown of alertsBreakdown;"
+      class="alerts-breakdown-type"
+      [ngClass]="[breakdown.type === focusType ? 'focus' : '']"
+      (click)="onToggleFocusType.emit(breakdown.type)"
```

Comment:
Anything that has click handler is a button. Please use correct semantic tags. It will be easier when we want to work on a11y.

Reply:
There are other clickable things in the DebuggerV2 UI that are not buttons. I added a TODO item here to address that in a batch fashion in the future.

Reply:
ACK.